tests/test_maintenance_tracker_api.py

```diff
@@ -0,0 +1,110 @@
+"""
+Test class for our Maintenance Tracker API endpoints
+"""
+import os
+import unittest
+from app import create_app
+
+class MaintenanceTrackerTestCase(unittest.TestCase):
+	"""
+	Class for our maintenance tracker tests
+	"""
+	def setUp(self):
+		"""
+		Initialize our variable before test
+		"""
+		self.app = create_app(config_name="testing")
+		self.client = self.app.test_client
+
+	def test_api_can_create_user_account(self):
+		"""
+		Test API can create user account
+		"""
+		res = self.client().post("/users/api/v1.0/account/register/", json=dict(
+			firstname="Edwin",
+			lastname="Lomolo",
+			email="edwin@gmail.com",
+			password=1234,
+			confirm_password=1234
+		))
+		self.assertEqual(res.status_code, 201)
+		self.assertTrue(str(res.data))
+
+	def test_api_can_log_in_user(self):
+		"""
+		Test API can login user
+		"""
+		res = self.client().post("/users/api/v1.0/authenticate/", json=dict(
+			email="milly@gmail.com",
+			password=4747
+		))
+		self.assertEqual(res.status_code, 200)
+		self.assertTrue(str(res.data))
```

Comment:
Normally `assertTrue` is vague as it can be misleading. Try to test if specific data can be found in the `res.data` by using methods like `assertIn` or `assertContains`